src/assets/i18n/en_US.json

```diff
@@ -360,9 +360,15 @@
   "account.punchout.configuration.link": "Configuration",
   "account.punchout.configuration.option.none.label": "None",
   "account.punchout.configuration.save_success.message": "The OCI punchout configuration has been saved.",
+  "account.punchout.configure.link": "Configure {{0}}",
   "account.punchout.create.description": "Please specify your user with your {{0, translate, account.punchout.type.text}} punchout credentials.",
-  "account.punchout.create.heading": "Create new {{0, translate, account.punchout.type.text}} punchout user",
-  "account.punchout.create.link": "Create new punchout user",
+  "account.punchout.create.heading": "Create a new {{0, translate, account.punchout.type.text}} punchout user",
+  "account.punchout.create.link": "Create a new punchout user",
+  "account.punchout.cxml.configuration.default.description": "Default value: {{defaultValue}}",
+  "account.punchout.cxml.configuration.helptext": "Use the following preferences to determine the content of the cXML Punchout document. Set values can be reset to return to the default value as specified.",
+  "account.punchout.cxml.configuration.link": "Configuration",
+  "account.punchout.cxml.configuration.no_configuration": "There is no configuration available at this time.",
+  "account.punchout.cxml.configuration.save_success.message": "The cXML Punchout configuration has been saved.",
   "account.punchout.cxml.info.url.helptext": "Please use the following URL for the punchout configuration in your cXML procurement system. For the sender credentials use the username as identity and the password as SharedSecret.",
```

Comment:
Identity and SharedSecret are in quotation marks in the other languages. Not needed in EN?